Quantum-Control-Applications-QuAM/Superconducting/README.md

```diff
@@ -0,0 +1,448 @@
+   opx_output = MWFEMAnalogOutputPort(
+      controller_id="con1", fem_id=1, port_id=2, band=1, upconverter_frequency=int(3e9), full_scale_power_dbm=-14
+   ),
+   upconverter=1,
+   intermediate_frequency=20e6
+)
+mw_in = InOutMWChannel(
+   id = "mw_in",
+   operations = {
+      "readout": SquareReadoutPulse(amplitude=0.1, length=100), },
+   opx_output = MWFEMAnalogOutputPort(
+      controller_id="con1", fem_id=1, port_id=1, band=1, upconverter_frequency=int(3e9), full_scale_power_dbm=-14
+   ),
+   opx_input = MWFEMAnalogInputPort(
+      controller_id="con1", fem_id=1, port_id=1, band=1, downconverter_frequency=int(3e9)
+   ),
+   upconverter=1,
+   time_of_flight=28,
+   intermediate_frequency=10e6
+)
+
+machine.qubits["dummy_out"] = mw_out
+machine.qubits["dummy_in"] = mw_in
+
+with program() as prog:
+   mw_out.play("cw")
+   mw_in.align()
+   mw_in.play("readout")
+
+config = machine.generate_config()
+qmm = machine.connect()
+
+simulation_config = SimulationConfig(duration=250)  # In clock cycles = 4ns
+job = qmm.simulate(config, prog, simulation_config)
+job.get_simulated_samples().con1.plot()
+
+# save machine into state.json
+machine.save("dummy_state.json")
+
+# %%
+# View the corresponding "raw-QUA" config
+with open("dummy_qua_config.json", "w+") as f:
+   json.dump(machine.generate_config(), f, indent=4)
+```
+### [4. Updating the parameters of state.json](./configuration/modify_quam.py)
+Once the state is created, each parameter can be updated based on the desired initial values using
+[modify_quam.py](./configuration/modify_quam.py).
+
+````python
+# %%
+import numpy as np
+import json
+from quam_libs.components import QuAM
+from quam_libs.quam_builder.machine import save_machine
+
+# Load QuAM
+path = "./quam_state"
+machine = QuAM.load(path)
+
+# %%
+# Update the resonator parameters
+rr_freq = np.array([4.395, 4.412, 4.521, 4.728, 4.915, 5.147]) * 1e9
+rr_LO = 4.75e9
+rr_if = rr_freq - rr_LO
+rr_max_power_dBm = -8
+
+for i, q in enumerate(machine.qubits):
+   machine.qubits[q].resonator.opx_output.full_scale_power_dbm = rr_max_power_dBm
+   machine.qubits[q].resonator.opx_output.upconverter_frequency = rr_LO
+   machine.qubits[q].resonator.opx_input.downconverter_frequency = rr_LO
+   machine.qubits[q].resonator.intermediate_frequency = rr_if[i]
+
+# %%
+# save into state.json
+save_machine(machine, path)
+
+# %%
+# View the corresponding "raw-QUA" config
+with open("qua_config.json", "w+") as f:
+   json.dump(machine.generate_config(), f, indent=4)
+````
+
+Note that these parameters serve as a starting point before starting to calibrate the chip and their values will be
+updated at the end of each calibration node.
+
+## How to run Qualibrate nodes
+### Node structure
+> **_NOTE:_**  For the most detailed and up-to-date documentation on calibration nodes, visit the QUAlibrate [documentation](https://qua-platform.github.io/qualibrate/calibration_nodes/).
+> 
```

Comment:
Instead I would rephrase it as `"For more details about Qualibrate and calibration nodes, visit the QUAlibrate [documentation](https://qua-platform.github.io/qualibrate/calibration_nodes/)."` to avoid leading the customers to think that the Github readmes are not up to date